dask/bytes/tests/test_s3.py

```diff
@@ -480,6 +480,60 @@ def test_parquet(s3, engine, s3so, metadata_file):
     tm.assert_frame_equal(data, df2.compute())
 
 
+@pytest.mark.parametrize("engine", ["pyarrow", "fastparquet"])
+def test_parquet_append(s3, engine, s3so):
+    pytest.importorskip(engine)
+    dd = pytest.importorskip("dask.dataframe")
+    pd = pytest.importorskip("pandas")
+    np = pytest.importorskip("numpy")
+    from dask.dataframe._compat import tm
```

Comment:
This seems like an odd `import` since `tm` is a pure-`pandas` import in the `_compat.py` module

https://github.com/dask/dask/blob/892ff1fc5cc58db4ad3af8e46de41dc41f408651/dask/dataframe/_compat.py#L16

Could we just use `pd.testing.assert_frame_equal` below instead?